docs/admin/backup.rst

```diff
@@ -0,0 +1,177 @@
+"Verify backup integrity, but do not restore the data." For details, see the 
+`QubesOS backup documentation <https://www.qubes-os.org/doc/backup-restore/>`_.
+
+Restore
+-------
+
+Reinstall QubesOS
+~~~~~~~~~~~~~~~~~
+
+To restore SecureDrop Workstation, follow our 
+:doc:`pre-install tasks <install>` to provision a QubesOS system complete with
+updated base templates.
+
+.. note::
+ If you backed up a number of custom VMs, during the installer, you may opt
+ to un-select "create default application qubes (personal, work, untrusted, vault)".
+
+Manage Unused VMs
+~~~~~~~~~~~~~~~~~
+
+If you installed all default application VMs, remove the ``vault`` VM, since we
+will replace it with the version in our backup.
+
+  .. code-block:: sh
+
+   qvm-prefs --set vault installed_by_rpm False 
+   qvm-remove vault
+
+Restore Backup
+~~~~~~~~~~~~~~
+
+Plug in your backup medium and unlock it as during the backup. By default
+on a new system, your peripheral devices will be managed by ``sys-usb``. 
+(For non-LUKS-based encryption media, you will need to install and configure the
+appropriate software, either in ``sys-usb`` or in another VM).
+
+Navigate to **Q ▸ System Tools ▸ Restore Backup**, and enter the  
+location of the backup file. You do not need to adjust the default Restore 
+options, unless you have made customizations to the backup. Enter the
+decryption/verification passphrase, and proceed to restore the backup.
+
+Your ``vault`` VM will be restored. 
+
+The contents of the previous ``dom0`` will be backed up to a folder in the 
+new ``dom0`` home directory called ``home-restore-<timestamp>/dom0-home/user``.
+In this guide we will refer to that location as ``$DOM0_BACKUP``.
+
+Open a terminal via **Q ▸ Terminal Emulator**: 
+
+  .. code-block:: sh
+
+   cd $DOM0_BACKUP/backup
+   cp -r securedrop-workstation-dom0-config/ /usr/share/
+
+If you customized clipboard or RPC policies, you may inspect the files in
+``backup/etc/qubes-rpc/`` and ``backup/etc/qubes/`` to ensure that the
+configurations in those files are re-applied to your new instance. 
+
+.. note::
+ Proceed with caution if overwriting files in ``/etc/qubes/`` and 
+ ``/etc/qubes-rpc/`` with their counterparts from your backup file, since this
+ can have unintended consequences, in particular if there are configuration 
+ differences between your old and new installation.   
+
+Reinstall SecureDrop Workstation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Follow the :doc:`installation instructions <install>` to download and verify
+the SecureDrop Workstation rpm to a network-attached VM and copy it to ``dom0``.
+
+.. note::
+ If your installation is recent, your backup archive may contain the current
+ version of the rpm, in which case you can skip the above steps and move
+ straight to installing the rpm. 
+ Use ``rpm -qi securedrop-workstation.rpm`` to compare the version with the
+ latest version at ``https://yum.securedrop.org/workstation/dom0/f25/``    
```

Comment:
In the interest of reducing `if`s—which I find cause me to stop reading and think hard about whether the condition applies to my use case—let's just recommend installing from scratch again. It feels like an edge case for the back up and restore to happen on the same version of `securedrop-workstation-dom0-config`. Even in the case of a minor version bump, we want to make sure the new system uses the latest and greatest.